liquid-compiler/src/filter_chain.rs

```diff
@@ -80,7 +31,7 @@ impl FilterChain {
 
         // apply all specified filters
         for filter in &self.filters {
-            entry = filter.evaluate(context, &entry)?;
+            entry = filter.filter(&entry, context)?;
```

Comment:
Maybe we should keep the name `evaluate` to align with the other `evaluate`s in liquid.

Thoughts?